core/src/core-api/splitViewHandle.js

```diff
@@ -0,0 +1,178 @@
+import { helpers } from './helpers';
+import { LuigiCoreAPIBase } from './baseClass';
+
+/**
+ * Split view 
+  Allows to open a micro frontend in a split screen in the lower part of the content area. Open it by calling `const splitViewHandle = Luigi.navigate().openAsSplitView`. 
+  At a given time, you can open only one split view. It closes automatically when you navigate to a different route.
+  When you call `handle.collapse()`, the split view gets destroyed. It recreates when you use `handle.expand()`.
+  `openAsSplitView` returns an instance of the split view handle. The functions, actions, and event handlers listed below allow you to control and manage the split view.
+  * @name splitView
+  * @since 0.6.0
```

Comment:
Please update all `@since` occurrences, because it will get into care api earliest with `0.7.4`. 

Just thinking about this, since the implementer does not know when it gets merged/released, any defined number might be wrong. We should add something like `* @since NEXTRELEASE`  .. and releasemeister will update it with the latest release, once it gets released.

Reply:
I've updated `@since` occurrences on the new functions I've added according to your suggestion. This comment also made me realise that this particular file was redundant and I had forgotten to delete it :) . (which also fixed unit test coverage issue)